ichnaea/conf.py

```diff
@@ -0,0 +1,117 @@
+"""
+Application settings and configuration derived from the environment
+"""
+from __future__ import absolute_import
+
+import os
+import os.path
+
+from everett.component import ConfigOptions, RequiredConfigMixin
+from everett.manager import ConfigManager, ConfigOSEnv
+
+HERE = os.path.dirname(__file__)
+
+
+class AppConfig(RequiredConfigMixin):
+    required_config = ConfigOptions()
+    required_config.add_option(
+        'testing',
+        default='false',
+        parser=bool,
+        doc='Whether or not we are running tests.'
+    )
+
+    required_config.add_option(
+        'asset_bucket',
+        default='',
+        doc='???'
+    )
+    required_config.add_option(
+        'asset_url',
+        default='',
+        doc='???'
+    )
+
+    # Database related settings
+    required_config.add_option(
+        'db_library',
+        default='pymysql',
+        doc='one of "pymysql" or "mysqlconnector"'
+    )
+    required_config.add_option(
+        'db_readonly_uri',
```

Comment:
"ro" and "rw" seemed too short and I know I read it wrong once or twice when skimming the code. For that reason, I expanded it to "readonly" and "readwrite".

I'm not sold on those settings names. I'm game for something that's clearer.